pkg/cmd/roachtest/roachtestutil/disk_usage.go

```diff
@@ -128,40 +129,41 @@ func NewDiskUsageTracker(
 	return &DiskUsageTracker{c: c, l: diskLogger}, nil
 }
 
-// GetDiskUsageInBytes does what's on the tin. nodeIdx starts at one.
```

Comment:
Let's add a description... something of the form, "GetDiskUsageInBytes executes the command `du {store-dir}`. Optional `args` are appended."